bin/s2nd.c

```diff
@@ -311,6 +353,12 @@ struct conn_settings {
     int max_conns;
     const char *ca_dir;
     const char *ca_file;
+    size_t psk_identity_count;
+    char psk_identity_list[S2N_MAX_NO_OF_PSKS_IN_LIST][UINT16_MAX];
+    size_t psk_secret_count;
+    char psk_secret_list[S2N_MAX_NO_OF_PSKS_IN_LIST][UINT16_MAX];
+    size_t psk_hmac_count;
+    char psk_hmac_list[S2N_MAX_NO_OF_PSKS_IN_LIST][UINT16_MAX];
```

Comment:
How large is UINT16_MAX? Is this a reasonable amount of stack memory?

Could you append PSKs as you receive them from the command line?